### DISCUSSION TITLE
框架开发的问题

### DISCUSSION THREAD

**Discussion by zhengzhuangpro** [openkraken/kraken#302] [Show and tell]
我用rax开发，Slider在手机上超级不好用，卡顿，基本是不能用，官方建议使用rax开发吗，那轮播组建有啥建议吗

**Comment by cnryb** [comment]
我看文档上介绍，北海是要支持W3C标准的，所以理论上北海不挑前端框架。

**Comment by wssgcg1213** [comment]
rax-slider 我们试了下确实没有那么好用  
可以尝试给 rax-slider 提 PR, 或者使用任意第三方的组件, 亦或通过 W3C 标准实现一个

https://github.com/raxjs/rax-components/tree/master/packages/rax-slider
